sql/src/main/java/io/crate/metadata/Scalar.java

```diff
@@ -79,6 +80,8 @@
     public Symbol normalizeSymbol(Function symbol, TransactionContext transactionContext) {
         try {
             return evaluateIfLiterals(this, symbol);
+        } catch (UnsupportedFeatureException e) {
```

Comment:
I already added the comment in addition to the commit message. You added your GitHub comment above it. Btw, I find it very helpful to look inside commit messages through the IntelliJ's annotate function to add a bigger picture to changes.